k8s/statefulset.yaml

```diff
@@ -0,0 +1,153 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: flink-jobmanager
+spec:
+  serviceName: flink-jobmanager
+  replicas: 1
+  selector:
+    matchLabels:
+      app: flink
+      component: jobmanager
+  template:
+    metadata:
+      labels:
+        app: flink
+        component: jobmanager
+    spec:
+      securityContext:
+        fsGroup: 9999
+      containers:
+      - name: jobmanager
+        image: 281050902442.dkr.ecr.eu-central-1.amazonaws.com/hackathon-2021-stateful-flink-process/flink:1.15-SNAPSHOT
+        imagePullPolicy: Always
```

Comment:
Why do we need this? Doesn't this increase the time it takes for starting new pods?

Reply:
Removed